index.html

```diff
@@ -291,5 +291,29 @@ <h5>Skills</h5>
           </div>
         </article>
       </section>
-		</body>
+      <section class="contact con-fnt">
+        <div class="cont_frm_shapes-mob"></div>
+          <header class="con-bg-ls">
+          <h1 class="con-txt-hd">Contact me</h1>
+            <p class="con-txt-bd">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+          </p>
+          </header>
+          <form>
```

Comment:
- [ ] Kindly add  action and method attributes. Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
- [ ] You will need to open an account with formspree and add your link to the action so that you will be able to get the email in your mailbox.